packages/mds-web-sockets/client.ts

```diff
@@ -0,0 +1,50 @@
+import WebSocket from 'ws'
+import { VehicleEvent, Telemetry } from '@mds-core/mds-types'
+import log from '@mds-core/mds-logger'
+import { setWsHeartbeat, WebSocketBase } from 'ws-heartbeat/client'
+
+const url = 'ws://localhost:4001'
+
+const { env } = process
+
+const { TOKEN } = env
```

Comment:
The extra destructure takes an extra step to trace. Why no just `const { TOKEN } = process.env`